src/media/css/feed/shelves.styl

```diff
@@ -121,3 +150,32 @@ shelf_curve_color($color) {
 [data-shelf-landing-carrier="mtn"] {
     collection_background_colors($mtn_color);
 }
+
+@media $base-tablet {
+    [data-shelf-landing] {
+        .feed-landing-header-mobile {
+            display: flex;
+
+            h1 {
+                font-size: 30px;
+                font-weight: 100;
+                line-height: 40px;
+                top: -7px;
+            }
+        }
+        .feed-landing-header-desktop {
+            display: none;
+        }
+    }
+}
+
+@media $base-desktop {
+    [data-shelf-landing] {
+        .feed-landing-header-mobile {
+            display: none;
+        }
+        .feed-landing-header-desktop {
+            display: flex;
+        }
```

Comment:
Are the designs separate enough that we need to toggle two separate headers? That seems really wasteful.